Contents/Code/__init__.py

```diff
@@ -1,245 +1,140 @@
-FEEDS_LIST = "http://feeds.theplatform.com/ps/JSON/PortalService/2.2/getReleaseList?PID=%s&startIndex=1&endIndex=500&query=categoryIDs|%s&sortField=airdate&sortDescending=true&field=airdate&field=author&field=description&field=length&field=PID&field=thumbnailURL&field=title&contentCustomField=title&contentCustomField=Episode&contentCustomField=Season"
-DIRECT_FEED = "http://release.theplatform.com/content.select?format=SMIL&pid=%s&UserName=Unknown&Embedded=True&TrackBrowser=True&Tracking=True&TrackLocation=True"
-LOADCATS = { 
-	'full':["/Shows/"],
-	'how-to':["/How-To/"]
-	}
-RE_SEASON_TEST = Regex("Season")
-VIDEO_URL = 'http://www.hgtv.ca/video/?releasePID=%s'
+PREFIX = '/video/hgtvcanada'
 
-####################################################################################################
-def Start():
-
-	Plugin.AddPrefixHandler('/video/hgtvcanada', MainMenu, TITLE)
-
-	# setup the default viewgroups for the plugin	
-	Plugin.AddViewGroup("InfoList", viewMode="InfoList", mediaType="items")
-	Plugin.AddViewGroup("List", viewMode="List", mediaType="items")
-	Plugin.AddViewGroup("Details", viewMode="InfoList", mediaType="items")
+TITLE = 'HGTV Canada'
+ART = 'art-default.jpg'
+ICON = 'icon-default.png'
 
-	# Setup the default attributes for the ObjectContainer
-	ObjectContainer.title1 = TITLE
-	ObjectContainer.view_group = 'List'
-	
-	# Setup some basic things the plugin needs to know about
-	HTTP.CacheTime = CACHE_1HOUR
+FEED_URL = 'http://feed.theplatform.com/f/dtjsEC/EAlt6FfQ_kCX?count=true&byCategoryIDs=%s&startIndex=%s&endIndex=%s&sort=pubDate|desc'
+MOST_RECENT_ITEMS = 50
+VIDEO_URL_TEMPLATE = 'http://www.hgtv.ca/video/video.html?v=%s'
 
 ####################################################################################################
-def MainMenu():
-	oc = ObjectContainer(
-		objects = [
-			DirectoryObject(
-				key = Callback(LoadShowList, cats='full'),
-				title = 'Shows'
-			),
-			DirectoryObject(
-				key = Callback(LatestShows),
-				title = 'Latest Videos Posted'
-			),
-			DirectoryObject(
-				key = Callback(LoadShowList, cats='how-to'),
-				title = 'How To Videos'
-			),
-	] )
-
-	return oc
-
-	
-####################################################################################################
-def LoadShowList(cats):
-	oc = ObjectContainer()
-	
-	shows_with_seasons = {}
-	shows_without_seasons = {}
-
-	network = HGTV_PARAMS
-	content = JSON.ObjectFromURL(FEED_LIST % (network[0], network[1]))
-
-	for item in content['items']:
-		if WantedCats(item['parent'],cats):
-			title = item['fullTitle'].split('/')[2]
-			iid = item['ID']
-			thumb_url = item['thumbnailURL']
-			
-			# a couple of shows have "Full Episodes" instead of "Season [0-9]"
-			if RE_SEASON_TEST.search(item['fullTitle']) or "Full Episodes" in item['fullTitle']:
```

Comment:
Have they really cleaned up all the old metadata in their older feeds?  Is this sort of stuff no longer needed?  In the past there were a LOT of corner cases in order to get all the shows and seasons to show up ...